virtual/environment.go

```diff
@@ -899,3 +899,12 @@ func formatActorCacheKey(
 	dst = append(dst, []byte(actorID)...)
 	return dst
 }
+
+func pickServerForInvocation(references []types.ActorReference, create types.CreateIfNotExist) types.ActorReference {
+	// TODO: implement invokation strategies in 'create' e.g. memory-balanced, cpu-balanced, multi-invoke...
+	sort.Slice(references, func(i, j int) bool {
```

Comment:
I dont think this makes sense as the default because in most reasonable use cases the replicas will have the same memory usage. I would do something easy for now like random

Reply:
I'm curious to understand your perspective on actor replicas and their memory footprint. In cases where actor replicas implement different calls with varying data inputs, it is highly likely that their memory usage will be unbalanced. I would like to explore further and gain insight into the specific use-case you have in mind, where a "standard" behavior leads to a more balanced memory usage. Understanding this scenario would help me broaden my understanding and identify any potential strategies for achieving memory balance in such situations.